src/backoffice/settings/dashboards/health-check/views/health-check-group.element.ts

```diff
@@ -0,0 +1,204 @@
+import { customElement, property, state } from 'lit/decorators.js';
+
+import { UmbHealthCheckContext } from '../health-check.context';
+import { UmbHealthCheckDashboardContext } from '../health-check-dashboard.context';
+import {
+	HealthCheckAction,
+	HealthCheckGroupWithResult,
+	HealthCheckResource,
+	HealthCheckWithResult,
+	StatusResultType,
+} from '@umbraco-cms/backend-api';
+import { UmbLitElement } from '@umbraco-cms/element';
+import { tryExecuteAndNotify } from '@umbraco-cms/resources';
+import './health-check-action.element';
+
+@customElement('umb-dashboard-health-check-group')
+export class UmbDashboardHealthCheckGroupElement extends UmbLitElement {
+	static styles = [
+		UUITextStyles,
+		css`
+			uui-box {
+				margin-bottom: var(--uui-size-space-5);
+			}
+
+			p {
+				margin: 0;
+			}
+
+			.header {
+				display: flex;
+				justify-content: space-between;
+				align-items: center;
+			}
+
+			.check-results-wrapper {
+				margin-top: var(--uui-size-space-5);
+			}
+
+			.check-results-wrapper .check-result:not(:first-child) {
+				padding-top: var(--uui-size-space-5);
+				border-top: 1px solid var(--uui-color-divider-standalone);
+			}
+
+			.check-results-wrapper uui-button {
+				margin-block-start: 1em;
+			}
+
+			.check-result-description {
+				display: flex;
+			}
+
+			.check-result-description span {
+				width: 36px;
+			}
+
+			uui-icon {
+				vertical-align: sub;
+			}
+		`,
+	];
+
+	@property()
+	groupName!: string;
+
+	@state()
+	private _buttonState: UUIButtonState;
+
+	@state()
+	private _group?: HealthCheckGroupWithResult;
+
+	private _healthCheckContext?: UmbHealthCheckDashboardContext;
+
+	@state()
+	private _checks?: HealthCheckWithResult[] | null;
+
+	@state()
+	private _keyResults?: any;
+
+	private _api?: UmbHealthCheckContext;
+
+	constructor() {
+		super();
+		this.consumeContext<UmbHealthCheckDashboardContext>('umbHealthCheckDashboard', (instance) => {
+			this._healthCheckContext = instance;
+
+			this._api = this._healthCheckContext?.apis.get(this.groupName);
+
+			this._api?.getGroupChecks(this.groupName);
+
+			this._api?.checks.subscribe((checks) => {
+				this._checks = checks;
+				this._group = { name: this.groupName, checks: this._checks };
+			});
+
+			this._api?.results.subscribe((results) => {
+				this._keyResults = results;
+			});
+		});
+	}
+
+	private async _buttonHandler() {
+		this._buttonState = 'waiting';
+		this._api?.checkGroup(this.groupName);
+		this._buttonState = 'success';
+	}
+
+	private _onActionClick(action: HealthCheckAction) {
+		return tryExecuteAndNotify(this, HealthCheckResource.postHealthCheckExecuteAction({ requestBody: action }));
+	}
+
+	render() {
+		if (this._group) {
+			return html`
+				<div class="header">
+					<h2>${this._group.name}</h2>
+					<uui-button
+						label="Get checks"
+						color="positive"
+						look="primary"
+						.state="${this._buttonState}"
+						@click="${this._buttonHandler}">
+						Get checks
```

Comment:
Should this be called "Perform checks" or something? I don't remember the name in the old backoffice.

Reply:
It was called "Get group" but I like "Perform checks" a lot better.